src/run.rs

```diff
@@ -30,6 +30,26 @@ fn edit<P: AsRef<OsStr>>(path: P) -> Result<(), i32> {
   }
 }
 
+const EMPTY_JUSTFILE: &str = "# this is an empty justfile\n";
+
+pub fn init() -> Result<(), i32> {
+  let mut root = search::project_root(&env::current_dir().unwrap());
+
+  root.push("Justfile");
+
+  if root.exists() {
```

Comment:
This will miss justfiles with a different capitalization on case-sensitive filesystems. Ideally, we should re-use the functionality in `search::justfile()` which searches for a justfile, which looks like this:

```
  for entry in dir {
    let entry = entry.map_err(|io_error| SearchError::Io {
      io_error,
      directory: directory.to_owned(),
    })?;
    if let Some(name) = entry.file_name().to_str() {
      if name.eq_ignore_ascii_case(FILENAME) {
        candidates.push(entry.path());
      }
    }
  }
```

I think it could be pulled into its own function, perhaps `search::dir`, and reused, checking for `SearchError::NotFound`.